app/views/admin/application/show.html.erb

```diff
@@ -35,18 +35,24 @@ as well as a link to its edit page.
 <section class="main-content__body">
 
   <dl>
-    <% page.attributes.each do |attribute| %>
-      <dt class="govuk-!-margin-top-0 attribute-label" id="<%= attribute.name %>">
-      <%= t(
-        "helpers.label.#{resource_name}.#{attribute.name}",
-        default: page.resource.class.human_attribute_name(attribute.name),
-      ) %>
-      </dt>
-
-      <dd class="attribute-data attribute-data--<%=attribute.html_class%>"
-          > <%= render_field attribute, page: page %></dd>
+    <% page.attributes.each do |title, attributes| %>
+      <fieldset class="<%= "field-unit--nested" if title.present? %>">
+        <% if title.present? %>
+          <legend><%= t "helpers.label.#{page.resource_name}.#{title}", default: title %></legend>
+        <% end %>
+        <% attributes.each do |attribute| %>
+          <dt class="attribute-label" id="<%= attribute.name %>">
```

Comment:
Do we want the `govuk-!-margin-top-0` class in this `dt` as before?

Reply:
I've put it back in for completeness sake